test/sql/base.pg

```diff
@@ -227,6 +274,39 @@ SELECT results_eq(
     'Foo bar diff check re-insert a prevoiusly deleted row #2'
 );
 
+-- EDIT: Update to same value
+
+\set last_revision :last_revision + 1
+SELECT is(table_version.ver_create_revision('foo.bar same-val-update'),
+    :last_revision, 'Create foo.bar same-val-update revision');
+
+UPDATE foo.bar SET d1 = d1;
+
+SELECT ok(table_version.ver_complete_revision(),
+    'Complete foo.bar same-val-update revision');
+
+SELECT is_empty(
+    format($$
+        SELECT * FROM table_version.ver_get_foo_bar_diff(%1$s - 1, %1$s)
```

Comment:
This would have returned nothing before this change as well. How about actually checking the table_version.foo_bar_revision table for ensure no new versioned rows were created?

Reply:
Uhm, it actually doesn't:
```
$ git revert 830a7359b9fefb6262d1f210a71df2d54432139d
$ make && make check
test/sql/base.pg ..... 1/176 
# Failed test 61: "foo.bar diff check for same-val-update"
#     Unexpected records:
#         (U,1,"foo bar 1 (re-insert)")
#         (U,2,"foo bar 2 edit")
#         (U,5,"foo bar 5")
#         (U,6,"foo bar 6")
# Failed test 62: "No entry in tables_changed for same-val-update on foo.bar"
#     Unexpected records:
#         (1007,1)
```

Reply:
But good point about adding another test directly in table_version.foo_bar_revision table, in case we want to fix the `_diff` function

Reply:
I've added your suggested test with a483d50